extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -252,6 +253,27 @@ export const SignTransaction = () => {
     );
   }
 
+  if (!isHttpsDomain) {
+    return (
+      <ModalWrapper>
+        <WarningMessage
+          handleCloseClick={() => window.close()}
+          isActive
+          isHighAlert
+          header={t("WEBSITE CONNECTION IS NOT SECURE")}
+        >
+          <p>
+            <Trans domain={domain}>
+              The website <strong>{{ domain }}</strong> does not use an SSL
```

Comment:
this is a good pattern to use if you want to wrap a translated string in bold or italics or something like that